ScreenManager.c

```diff
@@ -153,7 +153,7 @@ static void checkRecalculation(ScreenManager* this, double* oldTime, int* sortTi
    *rescan = false;
 }
 
-static inline bool drawTab(int* y, int* x, int l, const char* name, bool cur) {
+static inline bool drawTab(const int* y, int* x, int l, const char* name, bool cur) {
```

Comment:
Shouldn't `x` be `const` too?

Reply:
All tab titles are in the same row, so `y` is const. But for writing the titles the row, `x`, has to be advanced.